src/utils.test.ts

```diff
@@ -0,0 +1,41 @@
+import { createUniqueKeyObject } from './utils';
+
+/**
+ * createUniqueKeyObject
+ */
+describe('createUniqueKeyObject', () => {
+  it('Returns an empty object for an empty input array', () => {
```

Comment:
please update test names to `should ...`

Reply:
test names updated with 'Should' word